connexion/utils.py

```diff
@@ -253,3 +254,8 @@ def ignore_aliases(self, *args):
     yaml.representer.SafeRepresenter.represent_scalar = my_represent_scalar
 
     return yaml.dump(openapi, allow_unicode=True, Dumper=NoAnchorDumper)
+
+
+def fullmatch(regex, string, flags=0):
+    """Emulate python-3.4 re.fullmatch()."""
```

Comment:
I was confused by this comment because python 3.4 is at End Of Life.
what about 
`backport of re.fullmatch() in python 3`